pg-dba/README.md

```diff
@@ -0,0 +1,45 @@
+# pg-dba Helm Chart
+
+
+## Chart Details
+
+This deploys a kubernetes job that runs [pg-dba](https://github.com/jonstacks/pg-dba)
+to help perform common database maintenance.
+
+## Installing the Chart
+
+To install the chart with the release name `my-pg-dba`:
+
+```bash
+$ helm install --name my-pg-dba c2fo-public/pg-dba
```

Comment:
Do we need to add the c2fo-public repo via Helm?

Reply:
Yes. I am going to add that on the main readme as soon as I publish our first public chart 😄